### ISSUE TITLE
Anytime the blue text link color `#2f7aba` is used against any background color other than white, it does not meet contrast requirements.  

### ISSUE THREAD

**Issue by McManning** [osuresearch/ui#89]
Anytime the blue text link color #2f7aba is used against any background color other than white, it does not meet contrast requirements.  
•	Table rows: Element has insufficient color contrast of 4.32 (foreground color: #2f7aba, background color: #f9f9f9, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
•	warning alert: Element has insufficient color contrast of 4.25 (foreground color: #2f7aba, background color: #f6f6f4, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
•	[WCAG 1.4.3 Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum)

**Comment by helloLilyX** [comment]
Do I want to just darken the color for all of our components that has a `theme="link"`? What about the native HTML <a> elements?
